docs/source/intro/tutorial.rst

```diff
@@ -570,6 +570,16 @@ Depending on which step the workflow is running, you should get something like t
 We can see that the ``MultiplyAddWorkChain`` is currently waiting for its *child process*, the ``ArithmeticAddCalculation``, to finish.
 Check the process list again for *all* processes (You should know how by now!).
 After about half a minute, all the processes should be in the ``Finished`` state.
+You can also see a nice overview of the steps the work chain with the ``verdi process status`` command:
+
+.. code-block:: console
+
+    $ verdi process status 19
+    MultiplyAddWorkChain<19> Finished [0] [3:result]
```

Comment:
One question here: what does the `[3:result]` represent? I see that it prints the `stepper_state_info` by digging a bit through the code, but it isn't clear to me what the stepper state is. 😅

Reply:
it follows the "steps through the outline" if you check the outline of the `MultiplyAddWorkChain` you will see there are three steps, the third of which is called `result`. The stepper_state_info also correctly takes while loops and if conditions into account